### ISSUE TITLE
Property Custom Rates

### ISSUE THREAD

**Issue by eirichmond** [iprosoftware/api-csharp-client#2]
Your Property Custom Rates is giving us an extra week commencing that the client can NOT edit in the CRM UI:
<img width="1354" alt="screen shot 2017-08-14 at 17 09 06" src="https://user-images.githubusercontent.com/525490/29306521-ff5a4d08-8195-11e7-98ff-e663ecbcb62f.png">

**Comment by ipro-andy** [comment]
Hi elliot, thanks for the feedback we are looking at solutions. The client should not be using this feature at the moment

**Comment by eirichmond** [comment]
No, but because the custom rates are in the source feed but the client can't edit the source feed and the dates are being fed into my logic which causes issues.

The client is editing what they think is the week commencing but in fact the week commencing in the feed is an extra pre week.

**Comment by eirichmond** [comment]
@ipro-andy this is still an ongoing issue for your client, 5 months old now - is it ever going to be actioned? I'm still having to hack what comes in via your feed! Seems ludicrous for me to keep hacking my logic to fix the source, it makes more sense to resolve the issue at the source.

**Comment by ipro-andy** [comment]
Hi elliot, we haven't had authorisation from mike to complete the work

**Comment by eirichmond** [comment]
@ipro-andy why do you need authorisation? It is a bug in your feed, the UI does not reflect the random week you are sending in the feed, therefore, it shouldn't require any authorisation to fix it is a necessity?

**Comment by ipro-andy** [comment]
It's not a bug, I remember specifically having to add it within the feed as without you couldn't create your pricing grids on the website.  However we do not have anything in the UI to edit those values. So there are two choices:

1/ Remove from the feed again

2/ Update the UI so that those odds days are editable within iPro and prices provided (this is quite difficult based on the format we have done)

**Comment by eirichmond** [comment]
But the random price doesn't reflect the previous week from the last month, its a completely random array with an odd price?
